Pokedex/Pokedex/PokemonTableViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  PokemonTableViewController.swift
+//  Pokedex
+//
+//  Created by Gladymir Philippe on 7/17/20.
+//  Copyright © 2020 Gladymir Philippe. All rights reserved.
+//
+
+import UIKit
+
+class PokemonTableViewController: UITableViewController {
+    
+    var pokemonController = PokemonController()
+    
+    var pokemon: Pokemon! {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.dataSource = self
+        tableView.delegate = self
```

Comment:
```suggestion
```
If you had a table view inside of a UIViewController then you would definitely need to set the delegate and datasource for the table view but since you are using a table view controller these are not needed.